README.md

```diff
@@ -94,6 +94,32 @@ Illustration of the methodology behind the Multimodal Universe. Domain scientist
 
 Please see the [Design Document](https://github.com/MultimodalUniverse/MultimodalUniverse/blob/main/DESIGN.md) for more context about the project. 
 
+## Citations & Acknowledgements
+
+If you make use of all or part of the Multimodal Universe dataset, please cite the individual datasets accordingly. The relevant BibTeX citations and text acknowledgement instructions for datasets can be generated through the [info.py](scripts/info.py) file (`python scripts/info.py --help` for details).
+
+It allows you to retrieve all of the dataset information, or just acknowledgement and citation information for some or all datasets. By not specifying a dataset, it will return all datasets. By not specifying at least one of `--citation` or `--acknowledge`, it will return all of the information (including license, homepage, etc.).
+```sh
+python scripts/info.py --cite --data <datasets>
+python scripts/info.py --acknowledge --data <datasets>
+```
+For example, to get the citations for the APOGEE and SDSS datasets and save them to `info_citation.txt`, run:
+```sh
+python scripts/info.py --cite --data apogee sdss -o info_citation.txt
+```
```

Comment:
Would it be preferable to have the output as a `.bib`? If it's this format anyways we may mention it in the example.

```suggestion
For example, to get the citations for the APOGEE and SDSS datasets and save them to `info_citation.bib`, run:
```sh
python scripts/info.py --cite --data apogee sdss -o info_citation.bib
```
```

Reply:
Yeah. The example should probably do that. (Encoding wise they are the same afaik).